recipes/liburing/all/conanfile.py

```diff
@@ -40,8 +42,11 @@ def requirements(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-        if Version(self.version) < "2.2":
+        if Version(self.version) < "2.2" or Version(self.version) >= "2.5":
             del self.options.with_libc
+        if Version(self.version) < "2.5":
+            del self.options.use_libc
+
```

Comment:
Assuming the above comment is applied, this can be reverted

Reply:
done